### PR TITLE
Add price feed configuration guidance

### PR DESCRIPTION
**Pull request by Reinis-FRP**
Signed-off-by: Reinis Martinsons <reinis@umaproject.org>

### CODE REVIEW

File: docs/developers/pf-configuration.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+* `tokenAddress`: Address of the per-share token balance that should be tracked within the pool.
+
+### TraderMade
+
+Select [TraderMade](https://github.com/UMAprotocol/protocol/blob/master/packages/financial-templates-lib/src/price-feed/TraderMadePriceFeed.js) price feed by setting `type: "tradermade"` in the configuration to fetch forex pricing data from TraderMade. Supported configuration parameters are listed below:
+
+* Common `minTimeBetweenUpdates` and `priceFeedDecimals` parameters.
+* `minuteLookback` (optional if `hourlyLookback` is provided): Number of seconds how far in the past the historical minute interval prices will be fetched. Maximum allowed value is 172800 seconds (2 days).
+* `hourlyLookback` (optional): Number of seconds how far in the past the historical hourly prices will be fetched. Hourly historical prices can be used as a fallback to the minute time-series if no minute data is available (e.g. on weekends). Maximum allowed value is 5184000 seconds (2 months).
+* `tradermadeApiKey`: This is an application specific parameter, where the user should provide its TraderMade account API key.
```
Comment (smb2796) [inline docs/developers/pf-configuration.md:105]: Think it is worth being more explicit that this should not be included in `DefaultPriceFeedConfigs.js`? Seems obvious, and I know you note that it is an application specific parameter, but thoughts on calling it out again?

**Review by smb2796 — APPROVED** [review]
Top notch. Left one kind of nit comment, but really have nothing to add. Great stuff

File: docs/developers/pf-configuration.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+Availability of price feed is an absolute must for liquidable contracts and they should also generally be used by any other contracts expecting quick settlement if price fetching is relatively straightforward. The only exceptions where implementing a price feed might not be practical are KPI options or tracking more complex index data requiring involved manual calculation.
+
+## Selecting a Price Feed
+
+Product developers should document their price feed implementation choice and configuration when writing UMIP for new price identifier or passing price feed configuration as ancillary data for a generic price identifier. [UMA protocol repo](https://github.com/UMAprotocol/protocol/tree/master/packages/financial-templates-lib/src/price-feed) contains number of price feed implementations and this documentation guides on selecting and configuring those that could be more generally applicable for most applications.
```
Comment (smb2796) [inline docs/developers/pf-configuration.md:12]: Is the plan to expand the instructions on setting ancillary data for the `TOKEN_PRICE` identifier once it is voted in? Or do you think that belongs in a different section or is unnecessary?

File: docs/developers/pf-configuration.md (hunk 1 above)
Comment (Reinis-FRP) [reply docs/developers/pf-configuration.md]: Added warning both to CW and TM

File: docs/developers/pf-configuration.md (hunk 2 above)
Comment (Reinis-FRP) [reply docs/developers/pf-configuration.md:12]: Will add the link to UMIP once its voted in.
Will merge now

**Pull request merged**